presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -163,6 +164,8 @@ public void setupClass()
     @AfterSuite
     public void tearDownClass()
     {
+        closeAllRuntimeException(functionAssertions);
```

Comment:
Do this after shutting down the executor, otherwise if this throws, executor won't be shutdown